crates/next-core/src/next_config.rs

```diff
@@ -115,6 +126,7 @@ pub enum RemotePatternProtocal {
 #[serde(rename_all = "camelCase")]
 pub struct ExperimentalConfig {
     pub server_components_external_packages: Option<Vec<String>>,
+    pub app_dir: Option<bool>,
```

Comment:
Wooo!

Reply:
Little change, but it actually broken the benchmarks since we need a next.config.js for the benchmarks now...